packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -31,6 +31,22 @@ interface JWTHeaders {
     iat: number
 }
 
+interface SlasJwtPayload {
```

Comment:
Looks like `jwt-decode` exports `JwtHeader` and `JwtPayload`. Can we use those, instead?

Reply:
SLAS has custom keys in the payload, but good suggestion, i've updated the code to extend from `JwtPayload`